src/careamics/config/architectures/lvae_model.py

```diff
@@ -0,0 +1,173 @@
+"""LVAE Pydantic model."""
+
+from typing import Literal
+
+from pydantic import ConfigDict, Field, field_validator, model_validator
+from typing_extensions import Self
+
+from .architecture_model import ArchitectureModel
+
+
+class LVAEModel(ArchitectureModel):
+    """LVAE model."""
+
+    model_config = ConfigDict(validate_assignment=True, validate_default=True)
+
+    architecture: Literal["LVAE"]
+    input_shape: int = Field(default=64, ge=8, le=1024)
+    multiscale_count: int = Field(default=5)  # TODO clarify
+    # 0 - off, len(z_dims) + 1
+    z_dims: list = Field(default=[128, 128, 128, 128])
+    output_channels: int = Field(default=1, ge=1)
+    encoder_n_filters: int = Field(default=64, ge=8, le=1024)
+    decoder_n_filters: int = Field(default=64, ge=8, le=1024)
+    encoder_dropout: float = Field(default=0.1, ge=0.0, le=0.9)
+    decoder_dropout: float = Field(default=0.1, ge=0.0, le=0.9)
+    nonlinearity: Literal[
+        "None", "Sigmoid", "Softmax", "Tanh", "ReLU", "LeakyReLU", "ELU"
+    ] = Field(
+        default="ELU",
+    )
+
+    predict_logvar: Literal[None, "global", "pixelwise", "channelwise"] = (
+        None  # TODO Remove ?
+    )
```

Comment:
If creation of likelihood models has been extracted to `likelihood_factory` then yes it seems this parameter can be removed from LVAE.